client/rest-high-level/src/main/java/org/elasticsearch/client/MLClient.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.client;
+
+public final class MLClient {
```

Comment:
not sure, but should this be MachineLearningClient?

Reply:
I considered this, as the IT is MLIT which is the nastiest name ever. But the spec shows them as `.ml.` so I wanted to stay true to it.. Do you consider renaming the class diverging, or is this ok? I could still do a `ml()` getter in the XPackClient so the `hlrc.xpack().ml()` would still work out.

Also, wow, i cant believe i did this... i need to remove the get's off those methods so they are not `hlrc.xpack().getML()` !!!!!

Reply:
good point about the get prefix, I had missed it. I don't mind leaving naming as-is.